README.md

```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a framework for creatign new projects in Ruby
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
Technically, the request gets parsed before any of the controller actions are run. Otherwise correct.